openedx/features/enterprise_support/api.py

```diff
@@ -233,6 +234,32 @@ def enterprise_enabled():
     return 'enterprise' in settings.INSTALLED_APPS and getattr(settings, 'ENABLE_ENTERPRISE_INTEGRATION', True)
 
 
+def enterprise_customer_for_request(request, tpa_hint=None):
```

Comment:
Fair points. Let's continue to discuss our approach for organizing enterprise code outside the context of this PR. Thanks for entertaining my questions.